mis_builder/models/mis_report_instance.py

```diff
@@ -310,8 +310,14 @@ def _get_filter_domain_from_context(self):
         for filter_name, values in mis_report_filters.items():
             if values:
                 value = values.get('value')
-                operator = values.get('operator', '=')
-                filters.append((filter_name, operator, value))
+                if not value:
+                    continue
+                if not isinstance(value, list):
+                    value = [value]
+                many_ids = self.report_instance_id.resolve_2many_commands(
+                    filter_name, value, ['id'])
+                for m in many_ids:
+                    filters.append((filter_name, 'in', [m['id']]))
```

Comment:
@apineux this part now ignores of the `operator`. I think it is important to preserve it because some extensions depend on it. I've added a few unit tests on the 10.0 branch to make sure the important parts are guaranteed. cc/ @benwillig 

I'm wondering if we should not handle a new operator here (named `all`?), since what is actually does is not supported by any of the standard Odoo operators.